### DISCUSSION TITLE
Implementing a plugin inspired by elasticsearch plugin code and licensing

### DISCUSSION THREAD

**Discussion by fmassot** [grafana/grafana#67927] [Data Sources]
Hi there,

I'm working on a plugin for [quickwit engine](https://github.com/quickwit-oss/quickwit). As Quickwit targets partial support of Elasticsearch API, I'm currently getting inspiration from the code base here and following ongoing changes on the Elasticsearch plugin. So first, thank you very much for sharing that here :)

As this repository is under AGPL v3, I will obviously publish our plugin under this license. I just want to reach you and check if this is fine for you guys.

**Comment by fmassot** [comment]
Here is the [repository](https://github.com/quickwit-oss/quickwit-datasource). This is still a WIP and will be available at the end of May 2023.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). This is phrased as a question, and we'd like questions about grafana to go to our[ community forum](https://community.grafana.com/). Feel free to make an [issue](https://github.com/grafana/grafana/issues) if you cannot find a solution and would like to request the feature.
